src/components/form/form.js

```diff
@@ -601,11 +671,19 @@ class Form extends React.Component {
    */
   formFooter = () => {
     const save = this.props.showSummary ? this.saveButtonWithSummary() : this.saveButton();
+    let padding = this.props.stickyFooterPadding;
+
+    if (padding && !padding.match(/px$/)) {
+      padding = `${padding}px`;
```

Comment:
No this is so they can tweak the padding to match the UI in it's none-sticky state - that could be anything.